liquibase-core/src/main/java/liquibase/change/core/AddAutoIncrementChange.java

```diff
@@ -51,7 +56,7 @@ public void setCatalogName(String catalogName) {
         this.catalogName = catalogName;
     }
 
-    @DatabaseChangeProperty(mustEqualExisting = "column.relation.schema")
+    @DatabaseChangeProperty(mustEqualExisting = "column.relation.schema", description = "Name of the database schema")
```

Comment:
Hey Adrian, I have another question for you. For example for this method, we are adding a description, but checking the published [documentation](https://docs.liquibase.com/change-types/add-auto-increment.html?Highlight=AddAuto) we already have the same description for it. What’s the reason for this?

```
@DatabaseChangeProperty(since = "3.0", mustEqualExisting = "column.relation.catalog", description = "Name of the database catalog")
public String getCatalogName() {
    return catalogName;
}
```

Reply:
I am not entirely sure, but my guess is that @adrian-velonis1 did this so that what is already in Flare (documentation) matches what is in the code. This way when they autogenerate the documentation, we aren't missing something that was manually added to Flare. 

Adrian, am I understanding this correctly?